src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,27 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
this values is won't change, we can move to class variable not to create new object per each method call

Reply:
Can you explain in more detail, please?